mautic/mautic
# Company merge

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/138
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR gives the ability to merge companies.

#### Steps to test this PR:
1. Apply this PR
2. Go to a company, click on the dropdown menu item "Merge company" and select a company you want to merge.
3. the resulting company should be a combination of the company you previously selected with the company selected from the modal window, it will redirect you to the resulting company (the one selected from the modal window.
4. The master company is the one selected from the modal window, so the empty fields on this company will be filled with values from the current company
